resources/locale.json

```diff
@@ -254,7 +254,20 @@
         "UI_PEACOCK_GAMECHANGERS_NEWZEALAND_NOBODIESHIDDEN_NAME": "Do not hide the bodies",
         "UI_PEACOCK_GAMECHANGERS_NEWZEALAND_NOBODIESHIDDEN_DESC": "Bodies must not be hidden. Once you hide a body, you will fail the mission.",
         "UI_PEACOCK_GAMECHANGERS_NEWZEALAND_CHANGEWEAPON_NAME": "Single Scope",
-        "UI_PEACOCK_GAMECHANGERS_NEWZEALAND_CHANGEWEAPON_DESC": "Your default sniper rifle has been changed to Jaeger 7 Covert now."
+        "UI_PEACOCK_GAMECHANGERS_NEWZEALAND_CHANGEWEAPON_DESC": "Your default sniper rifle has been changed to Jaeger 7 Covert now.",
+        "UI_MENU_PAGE_PROFILE_CHALLENGES_CATEGORY_PACK_CHEESECAKE": "Butcher's Delight",
+        "CHALLENGEPACK_CHEESECAKE_WRAPPER_NAME": "Gourmet Temptations",
+        "CHALLENGEPACK_CHEESECAKE_WRAPPER_DESC": "Complete all of the Butcher's Delight challenges. <li>Hone Your Cleaver</li><li>How do Bones Taste?</li><li>Marinate for Flavour</li><li>Premium Barbeque</li><li>Dinner's Ready!</li>",
+        "CHALLENGEPACK_CHEESECAKE_SAPIENZACLEAVERKILL_NAME": "Hone Your Cleaver",
+        "CHALLENGEPACK_CHEESECAKE_SAPIENZACLEAVERKILL_DESC": "<li>Eliminate a target with Cleaver disguised as Kitchen Assistant in Sapienza.</li>",
+        "CHALLENGEPACK_CHEESECAKE_SANTAFORTUNABONEPACIFY_NAME": "How do Snake Bone Taste?",
```

Comment:
This should be changed to `How do Bones Taste?`

Reply:
Was going to make a pun. Forget it.

Reply:
Ah, I was just going off what was in the wrapper description. I will push to your fork soon with some updated challenges.